src/Splunk.Client.Helpers/Splunk/Client/Helpers/SdkHelper.cs

```diff
@@ -39,7 +39,7 @@ static SdkHelper()
             //// 2. Set its ServerCertificateValidationCallback
             //// 3. Instantiate a Splunk.Client.Context with the WebRequestHandler
 
-            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls;
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
Does moving from Tls to Tls12 have any downsides?

Reply:
This is a test helper so no

Reply:
Ah.  Right.